orderer/consensus/smartbft/chain.go

```diff
@@ -167,6 +167,17 @@ func NewChain(
 		return nil, errors.Wrap(err, "failed to verify SmartBFT-Go configuration")
 	}
 
+	c.clusterService = &cluster.ClusterService{
```

Comment:
why are you re-defining this?

Reply:
Will be updated in a "post-review" commit as we discussed.